plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployCommandHandler.java

```diff
@@ -140,10 +138,17 @@ private static boolean checkProjectErrors(IProject project) throws CoreException
     return severity != IMarker.SEVERITY_ERROR;
   }
 
+  protected void onDeployJobCreation() {
```

Comment:
Could we make this method name less about the implementation? `onDeployStart()`?

Reply:
Makes sense.